contracts/test/DaiEchidnaTest.sol

```diff
@@ -59,6 +69,8 @@ contract DaiEchidnaTest {
 
     /// @dev Test that supply and balance hold on transferFrom.
     function transferFrom(uint256 wad) public {
+        wad = wad % uint256(-1);
```

Comment:
Similar comment--you don't want to try to transfer more than the holder has.

Reply:
Done.

Reply:
same as above for `transferFrom` test.

Reply:
Done.